frontend/src/component/layout/MainLayout/NavigationSidebar/NavigationList.tsx

```diff
@@ -121,26 +123,43 @@ export const OtherLinksList = () => {
 
 export const PrimaryNavigationList: FC<{
     mode: NavigationMode;
-    onClick?: () => void;
-}> = ({ mode, onClick }) => {
+    onClick: (activeItem: string) => void;
+    activeItem?: string;
+}> = ({ mode, onClick, activeItem }) => {
     const DynamicListItem = mode === 'mini' ? MiniListItem : FullListItem;
 
     return (
         <List>
-            <DynamicListItem href='/projects' text='Projects' onClick={onClick}>
+            <DynamicListItem
+                href='/projects'
+                text='Projects'
+                onClick={() => onClick('/projects')}
+                selected={activeItem === '/projects'}
```

Comment:
Yeah I guess this might be the most elegant way to go about this, to tell each component which route-parent is the current one. 🤔